src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String COMMA = ",";
+    private static final String RESULT = "result";
+    private int sumOfSupply = 0;
+    private int sumOfBuy = 0;
```

Comment:
These variables should be local to the `readDataFromFile` method instead of class fields since they represent temporary state during a single operation and not the state of the class object.